src/index.js

```diff
@@ -154,9 +161,13 @@ if (require.main === module) {
       debug('Setting up data', options.data);
       setup.push(addObjectsToData(options.data).then((result) => data = result));
     }
+    if (options.stdout) {
+      debug('disabling console out');
+      console.log = function empty() {};
+    }
```

Comment:
Rather than doing this, we should change all calls to `console.log` to `console.err`, which outputs to stderr - this way we'll still get the necessary logs to terminal while being able to redirect stdout in the right way.